lms/djangoapps/experiments/utils.py

```diff
@@ -148,18 +243,34 @@ def get_experiment_user_metadata_context(course, user):
                 # A course can be in multiple programs, but we're just grabbing the first one
                 program = programs[0]
                 complete_enrollment = False
+                has_courses_left_to_purchase = False
                 total_courses = None
                 courses = program.get('courses')
+                courses_left_to_purchase_price = None
+                courses_left_to_purchase_url = None
                 if courses is not None:
                     total_courses = len(courses)
                     complete_enrollment = is_enrolled_in_all_courses_in_program(courses, user_enrollments)
 
+                    if PROGRAM_PRICE_FLAG.is_enabled():
+                        non_audit_enrollments = [enrollment for enrollment in user_enrollments if enrollment not in
+                                                 audit_enrollments]
+                        courses_left_to_purchase = get_unenrolled_courses_in_program(courses, non_audit_enrollments)
```

Comment:
courses_left_to_purchase looks like the right variable but i think the logic is still getting the courses the learner is not enrolled in?

Reply:
Note to self: rename method